### ISSUE TITLE
Navigation view animation issue

### ISSUE THREAD

**Issue by thisIsTheFoxe** [CSolanaM/SkeletonUI#54]
**Describe the bug**
When trying to use SkeletonUI in a NavStack it seems to animate the view from the origin. 

**To Reproduce**
```swift
import SwiftUI
import SkeletonUI

struct SkeletionAnimationView: View {
    var body: some View {
        NavigationStack {
            Text("")
                .skeleton(with: true)
        }
    }
}

#Preview {
    SkeletionAnimationView()
}
```

**Expected behavior**
Same as w/o the NavStack. It should show a fullscreen capsule with the skeleton animation. 

**Screenshots**
https://github.com/CSolanaM/SkeletonUI/assets/18512366/8a5a8e03-eea2-4ce7-8012-79766b7ac997

**Smartphone:**
 - OS: iOS 17.4.1
 - Version 2.0.1

**Additional context**
Don't want to assume anything, but possibly an iOS issue: 
- [rdar](rdar://problem/FB13795514)://[FB13795514](https://openradar.appspot.com/FB13795514)

**Comment by thisIsTheFoxe** [comment]
Just saw your comment about using `.main` queue in #52, using that in the `SkeletonModifier` seems to fix that issue.

**Comment by CSolanaM** [comment]
Thanks for that! I was skeptical if this was being reproduced by other people and if what I suggested would work in every case. Looks like I can release a new version with the fix! Thanks @thisIsTheFoxe !

**Comment by CSolanaM** [comment]
Fixed in 2.0.2
